lib/nibss/README.md

```diff
@@ -1,42 +1,54 @@
 # INNOVATION SANDBOX
 
 ## Install
+
 ```bash
 $ npm install innovation-sandbox
 ```
 
 ## Require NIBSS Library
+
 ```javascript
-const { nibss } = require('innovation-sandbox')
+const { nibss } = require("innovation-sandbox");
 ```
 
-## RESET_TOKEN([0ptions])
+## RESET_TOKEN([Options])
+
 This should be done on every initialization.
 
 ### Options
+
 The module accepts options as objects of key-value and returns aes_key, ivkey and password .
 
 #### sandbox_key
-This can be found in the innovation sandbox dashboard after signup. However `0ae0db703c04119b3db7a03d7f854c13` can be used for testing purposes.
+
+This can be found in the innovation sandbox dashboard after sign up. Register on `https://sandbox.fsi.ng/`
 
 #### organisation_code
-Credential is gotten from nibss. prior to acquiring this credential, 11111 should be used for testing rather documentation should say use organisation code from Dashboard.
+
+Credential is gotten from nibss. Prior to acquiring this credential, 11111 should be used for testing.
 
 ##### host
+
 This argument is optional in all cases. Defaults to `https://sandboxapi.fsi.ng` if not found.
 
 ### nibss.Bvnr.Reset(credentials)
-Returns the aes_key, ivey and password. As earlier stated this function should be called at every initialization. The credentials returned is set as part of the data for accessing other nibss modules.
+
+Returns the aes_key, ivey and password as part of the response header. As stated earlier this function should be called at every initialization. The credentials returned are set as part of the data for accessing other nibss modules.
 
 The following example calls the reset module
+
 ```javascript
-nibss.Bvnr.Reset({sandbox_key:'Your sandbox_key', organisation_code: 'Your organisation_code'})
-.then((response) => {
-    //do something with the response here
-})
+nibss.Bvnr.Reset({
+  sandbox_key: "Your sandbox_key",
+  organisation_code: "Your organisation_code" || "11111"
```

Comment:
"Your organisation_code" replace with "organisation_code from sandbox dashboard" and apply everywhere necessary

Reply:
Applied everywhere.